### PR TITLE
clojure: Update to version 1.9.0

### PR DESCRIPTION
**Pull request by alvinfrancis**
#### Description

This PR updates the clojure port to version 1.9.0 which now ships with CLI tools.  The port has been changed so as to mirror the official linux and homebrew installers in behaviour.

Closes: https://trac.macports.org/ticket/55848

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6 17G65
Xcode 9.4.1 9F2000

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: lang/clojure/Portfile [hunk 1]
```diff
@@ -4,8 +4,8 @@ PortSystem          1.0
 
 name                clojure
 conflicts           clojure-devel
-epoch               20110929
-version             1.8.0
+epoch               20180727
```
Comment (mf2k) [inline lang/clojure/Portfile]: Please do not change the value from what it was before. Just keep it at `20110929`.

File: lang/clojure/Portfile [hunk 2]
```diff
@@ -4,8 +4,7 @@ PortSystem          1.0
 
 name                clojure
 conflicts           clojure-devel
-epoch               20110929
```
[the remark this answers is not in the document]
Comment (alvinfrancis) [reply lang/clojure/Portfile]: `epoch` line re-added with new value.

[the remark this answers is not in the document]
Comment (alvinfrancis) [reply lang/clojure/Portfile]: Ah. Noted.

[the remark this answers is not in the document]
Comment (pmetzger) [reply lang/clojure/Portfile]: Why with a new value? That seems unreasonable.

**Comment by pmetzger** [comment]
@mf2k You want to commit or should I?

**Comment by mf2k** [comment]
Please go ahead if you think it's ready.

**Comment by pmetzger** [comment]
Merged. Thanks, @alvinfrancis!

**Pull request merged**